prow/tide/status_test.go

```diff
@@ -1303,7 +1317,13 @@ func TestStatusControllerSearch(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			ghc := &fgc{prs: tc.prs}
 			cfg := func() *config.Config {
-				return &config.Config{ProwConfig: config.ProwConfig{Tide: config.Tide{Queries: config.TideQueries{{Orgs: []string{"org-a", "org-b"}}}}}}
```

Comment:
What's this diff about?

Reply:
I added ``tc.maxGraphQLGoroutines`` and reformatted the code to have more readability. I find one liner to be quite difficult to read and edit.

Reply:
Easy to write, hard to review ;)